Disable gtk-update-icon-cache command at install script
beacuse xbps already triggers this on install and is
redundant.
diff -Naur a/install.sh b/install.sh
--- a/install.sh	2025-02-09 23:33:26.000000000 -0600
+++ b/install.sh	2025-12-09 22:48:09.417212095 -0600
@@ -283,8 +283,6 @@
   ln -sr "${THEME_DIR}/24"                                                       "${THEME_DIR}/24@2x"
   ln -sr "${THEME_DIR}/32"                                                       "${THEME_DIR}/32@2x"
   ln -sr "${THEME_DIR}/scalable"                                                 "${THEME_DIR}/scalable@2x"
-
-  gtk-update-icon-cache "${THEME_DIR}"
 }
 
 uninstall_theme() {
